packages/next/src/client/components/router-reducer/fetch-server-response.ts

```diff
@@ -185,7 +185,7 @@ export async function fetchServerResponse(
     const canonicalUrl = res.redirected ? responseUrl : undefined
 
     const contentType = res.headers.get('content-type') || ''
-    const interception = !!res.headers.get('vary')?.includes(NEXT_URL)
```

Comment:
let's keep this unchanged in case user appends additional vary headers to a response